JS/Airport.js

```diff
@@ -1,67 +1,54 @@
 const PassengerPlane = require('./Planes/PassengerPlane');
 const MilitaryPlane = require('./Planes/MilitaryPlane');
-const MilitaryType = require('./models/militaryType');
-const experimentalPlane = require('./Planes/experimentalPlane');
+const MilitaryType = require('./models/MilitaryType');
+const ExperimentalPlane = require('./Planes/ExperimentalPlane');
 
 class Airport {
-
-     getPasPl() {
-        let pl = this.planes;
-        var x = [];
-        for (let p of this.planes) {
-            if (p instanceof PassengerPlane) {x.push(p);}
+     getPassengerPlanes() {
+        var passengerPlaneList = [];
+        for (let plane of this.planes) {
+            if (plane instanceof PassengerPlane) {passengerPlaneList.push(plane);}
         }
-        return x;
+        return passengerPlaneList;
     }
 
     getMilitaryPlanes() {
-        let militaryPlanes = [];
+        let militaryPlanesList = [];
         this.planes.forEach(plane => {
-            if (plane instanceof MilitaryPlane) {//if
-                militaryPlanes.push(plane);
+            if (plane instanceof MilitaryPlane) {
+                militaryPlanesList.push(plane);
             }
-            //else
-            else{}
         });
-//return
-        return militaryPlanes;
+        return militaryPlanesList;
     }
 
     getPassengerPlaneWithMaxPassengersCapacity() {
-        let passengerPlanes = this.getPasPl();
+        let passengerPlanes = this.getPassengerPlanes();
         let planeWithMaxCapacity = passengerPlanes[0];
         for (let i = 0; i < passengerPlanes.length; i++) {
-            if (passengerPlanes[i].getPassengersCapacity() >     planeWithMaxCapacity.getPassengersCapacity()) {
+            if (passengerPlanes[i].getPassengersCapacity() > planeWithMaxCapacity.getPassengersCapacity()) {
                 planeWithMaxCapacity = passengerPlanes[i];
             }
         }
         return planeWithMaxCapacity;
     }
 
-
-
-
-
-
-
     getTransportMilitaryPlanes(){
         let transportMilitaryPlanes = [];
         let militaryPlanes = this.getMilitaryPlanes();
         for (let i = 0; i < militaryPlanes.length; i++) {
-        if (militaryPlanes[i].getMilitaryType() == MilitaryType.TYPE_TRANSPORT) {
+        if (militaryPlanes[i].getMilitaryType() == MilitaryType.TRANSPORT) {
         transportMilitaryPlanes.push(militaryPlanes[i]);
-        }
-        }
+        }}
         return transportMilitaryPlanes;
     }
 
-
     getBomberMilitaryPlanes()
     {
         let bomberMilitaryPlanes = [];
         let militaryPlanes = this.getMilitaryPlanes();
         for (let i = 0; i < militaryPlanes.length; i++) {
```

Comment:
Try to use `filter()` array method for the purpose of this code.

Reply:
Done